### PR TITLE
net/wireguard: add DNS to server config

### PR DESCRIPTION
**Pull request by mimugmail**
Add DNS servers to interface via `resolvconf`
Now local DNS lookups work with AzireVPN.

No bump since 0.4 isn't released yet.

### CODE REVIEW

File: net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml [hunk 1]
```diff
@@ -31,6 +31,9 @@
                     <default>51820</default>
                     <Required>Y</Required>
                 </port>
+                <dns type="CSVListField">
```
Comment (fichtner) [inline net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml]: No validation?

Comment (mimugmail) [reply net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml]: Hm, wasn't this the thing with v4 and v6 that doesn't Work?

Comment (mimugmail) [reply net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml]: Took the one from Zabbix Agent and removed the hostname stuff.

Comment (fichtner) [reply net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml]: Ok, thanks! Please talk to @AdSchellevis about finishing the built-in validation or how to use it.

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**